.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,33 @@
+name: "\U00002753 Question"
```

Comment:
Can we direct questions to the discussion arra in github?

Reply:
I'm not sure I understand you.
Did you mean to direct the person to open a discussion instead of an issue, or convert an issue into a discussion?

Reply:
Which ever is easier, I don't want to have issues that are questions, for that we have the discussions tab here (which probably no one will use, but should in the far far future be a knowledge database, again, in theory).
This one I mean:
https://github.com/IsraelHikingMap/Site/discussions